CHANGELOG.md

```diff
@@ -57,6 +57,7 @@
 * Added `RoadObjectsStore.addUserDefinedRoadObject(_:)`, `RoadObjectsStore.removeUserDefinedRoadObject(identifier:)`, and `RoadObjectsStore.removeAllUserDefinedRoadObjects()` for adding/removing user-defined road objects to the electronic horizon. ([#3004](https://github.com/mapbox/mapbox-navigation-ios/pull/3004))
 * Removed `Alert` enum, and `alert`, `distance`, `length`, `beginCoordinate`, `endCoordinate`, `beginSegmentIndex`, and `endSegmentIndex` properties from `RouteAlerts`. ([#2991](https://github.com/mapbox/mapbox-navigation-ios/pull/2991))
 * Added the `RouteAlerts.roadObject` property. ([#2991](https://github.com/mapbox/mapbox-navigation-ios/pull/2991))
+* Renamed `RoadObjectsStore` and `RoadObjectsStoreDelegate` to `RoadObjectStore` and `RoadObjectStoreDelegate` respectively. ([#3045](https://github.com/mapbox/mapbox-navigation-ios/pull/3045))
```

Comment:
RoadObjectsStore and RoadObjectsStoreDelegate were first introduced in a previous v2.0.0 beta, so we would mention this rename in the next beta’s release notes but not this running changelog.

Reply:
I see. I'll remove this one. And should we also update the name of the `RoadObjectsStore` and `RoadObjectsStoreDelegate` to `RoadObjectsStore` and `RoadObjectsStoreDelegate` in this changeLog list?